backend/danswer/configs/constants.py

```diff
@@ -12,7 +12,7 @@
 ALLOWED_USERS = "allowed_users"
 ALLOWED_GROUPS = "allowed_groups"
 METADATA = "metadata"
-OPENAI_API_KEY_STORAGE_KEY = "openai_api_key"
+GEN_AI_API_KEY_STORAGE_KEY = "genai_api_key"
```

Comment:
will force everyone to re-enter their openai key (which they may not have saved) :( 

Probably fine, but they may have to generate / ask for a new one

Reply:
sucks but whatever, probably not many people impacted anyhow